runtime/jcl/common/reflecthelp.c

```diff
@@ -759,6 +759,9 @@ createField(struct J9VMThread *vmThread, jfieldID fieldID)
 	j9object_t fieldObject = NULL;
 	J9Class *jlrFieldClass = J9VMJAVALANGREFLECTFIELD(vmThread->javaVM);
 	UDATA initStatus;
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+	U_32 valueTypeFlags = 0; /* used to calculate value of Field.flags in value type builds */
```

Comment:
I slightly prefer using the name "fieldFlags", because `TRUST_FINAL` can be applied to non-value types and the JCL change to `Field.flags` might eventually be merged into non-valhalla Jdknext repo.